appveyor.yml

```diff
@@ -0,0 +1,50 @@
+version: 4.1.{build}
+
+image:
+- Visual Studio 2015
+
+environment:
+  matrix:
+    - platform: x86
+      ARCH: x86
+      GENERATOR: "Visual Studio 14 2015"
+      PYTHON_VER: Python27
+    - platform: x86
+      ARCH: x86
+      GENERATOR: "Visual Studio 14 2015"
+      PYTHON_VER: Python34
+    - platform: x64
+      ARCH: amd64
+      GENERATOR: "Visual Studio 14 2015 Win64"
+      PYTHON_VER: Python27-x64
+    - platform: x64
+      ARCH: amd64
+      GENERATOR: "Visual Studio 14 2015 Win64"
+      PYTHON_VER: Python34-x64
+
+before_build:
+- git clone https://github.com/yaml/libyaml.git
+- call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %ARCH%
+- set PATH=C:\%PYTHON_VER%\Scripts;%PATH%
+- C:\%PYTHON_VER%\python.exe -m pip install cython wheel
+
+build_script:
+- mkdir %PYTHON_VER%
+- cd %PYTHON_VER%
+- cmake.exe -G"%GENERATOR%" -DBUILD_SHARED_LIBS=ON ..\libyaml
+- cmake.exe --build . --config Release --clean-first
+- cd ..
+- C:\%PYTHON_VER%\python.exe setup.py --with-libyaml build_ext -I C:\projects\pyyaml\libyaml\include -L C:\projects\pyyaml\%PYTHON_VER%\;C:\projects\pyyaml\%PYTHON_VER%\Release build bdist_wininst bdist_wheel
+
+after_build:
```

Comment:
Might be worth adding `twine upload`

Reply:
Interesting to know, but mostly we're trying to work out all the kinks in building
itself for the moment.

Reply:
Yeah, feel free to ping me once you get to upload, I've got some experience